### DISCUSSION TITLE
Is this usable with unplugin-vue-components

### DISCUSSION THREAD

**Discussion by Padda4** [LouisMazel/maz-ui#581] [Q&A]
Hi, 
i like maz ui and want to try it in a project but i cant get it to work with unplugin-vue-components.
[https://github.com/antfu/unplugin-vue-components](url)

Is there a way to use it with vite?

Cheers

**Comment by LouisMazel** [comment]
Hi @Padda4,

Solved in [v3.23.2](https://github.com/LouisMazel/maz-ui/releases/tag/v3.23.0)

Check the doc: https://louismazel.github.io/maz-ui-3/guide/getting-started#recommendations
